### PR TITLE
Content Guidelines

### PR DESCRIPTION
**Pull request by it33**
- Updating Marketing guidelines to "Community" guidelines and adding a note about editing and removing inappropriate material

### CODE REVIEW

File: source/process/marketing-guidelines.rst [hunk 1]
```diff
@@ -63,3 +63,23 @@ Guidelines: 12 tips for sparkling content
...
+
+
+Twitter Guidelines 
+==============================================================
+
+Composing Tweets: 
+
+1. **Avoid acronyms** - Avoid acronyms when possible. For example, say "pull request" instead of PR", since only a subset of Tweet readers will be active GitHub users. 
+2. **Use the active voice** - Avoid "has", "was", "have been" when possible. For example, instead of "Hackfest has started!" say "Hackfest starts now!" 
+3. **Include zero to one links, close to the end"** - Include no more than one link per tweet, and place it close to the end of the tweet.
```
Comment (esethna) [inline source/process/marketing-guidelines.rst]: Wording of the bold text is a bit confusing. Propose the bold text as `No more than one link, placed near the end`

File: source/process/marketing-guidelines.rst [hunk 2]
```diff
@@ -63,3 +63,23 @@ Guidelines: 12 tips for sparkling content
...
+2. **Use the active voice** - Avoid "has", "was", "have been" when possible. For example, instead of "Hackfest has started!" say "Hackfest starts now!" 
+3. **Include zero to one links, close to the end"** - Include no more than one link per tweet, and place it close to the end of the tweet. 
+
+
+Community Content Guidelines 
+==============================================================
+
+Content from community in GitHub, Forums, Feature Ideas, Community Mattermost site or any other community content site managed by Mattermost should follow Mattermost guidelines: 
+
+Content that is deemed to be controversial, offensive, potentially defamatory or infringing content from material distributed, broadcast, posted on websites will be edited or removed.
```
Comment (esethna) [inline source/process/marketing-guidelines.rst]: `broadcast, posted` > `broadcast, or posted`

File: source/process/marketing-guidelines.rst [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ============================================================
-Mattermost Marketing Content Guidelines
+Mattermost Community Content Guidelines
```
Comment (esethna) [inline source/process/marketing-guidelines.rst]: Not sure I agree with changing this to "community" guidelines? The entire doc except the last new section is about marketing.

File: source/process/marketing-guidelines.rst [hunk 4]
```diff
@@ -63,3 +63,23 @@ Guidelines: 12 tips for sparkling content
...
+==============================================================
+
+Composing Tweets: 
+
+1. **Avoid acronyms** - Avoid acronyms when possible. For example, say "pull request" instead of PR", since only a subset of Tweet readers will be active GitHub users. 
+2. **Use the active voice** - Avoid "has", "was", "have been" when possible. For example, instead of "Hackfest has started!" say "Hackfest starts now!" 
+3. **Include zero to one links, close to the end"** - Include no more than one link per tweet, and place it close to the end of the tweet. 
+
+
+Community Content Guidelines
```
Comment (jasonblais) [inline source/process/marketing-guidelines.rst]: Does this section fit here? I feel like we could add this to our internal community forum guidelines and make it public but it another doc

File: source/process/marketing-guidelines.rst (hunk 1 above)
Comment (it33) [inline source/process/marketing-guidelines.rst]: While I agree what's there isn't great, I don't know if `No more than one link, placed near the end` is much better.

File: source/process/marketing-guidelines.rst (hunk 3 above)
Comment (it33) [inline source/process/marketing-guidelines.rst]: See also: https://github.com/mattermost/docs/pull/716/commits/18dea8339b09d385fd20bd1d9cdbab70206f8e32

File: source/process/marketing-guidelines.rst [hunk 5]
```diff
@@ -1,7 +1,9 @@
 ============================================================
-Mattermost Marketing Content Guidelines
+Mattermost Community Content Guidelines
 ============================================================
 
+The following guidelines are designed to help community members contributing guest blog posts or proposing Twitter content, as well as for staff at Mattermost, Inc. engaging through community channels.
```
Comment (jasonblais) [inline source/process/marketing-guidelines.rst]: contributing --> contribute; 
proposing --> propose; 
engaging --> engage

File: source/process/marketing-guidelines.rst [hunk 6]
```diff
@@ -63,3 +65,15 @@ Guidelines: 12 tips for sparkling content
...
  11. **Ask a colleague**. Still not 100% happy? A colleague should review the text and tell you if anything is unclear or if you could express yourself better. They might offer some useful fresh insights.
  12. **Now, relax**. If you are in the wrong frame of mind you will either get writer's block or your writing will be stiff and unnatural. Take your time. Remove any distractions. Keep things simple. 
+
+
+Twitter Guidelines 
+==============================================================
+
+Composing Tweets: 
+
+1. **Avoid acronyms** - Avoid acronyms when possible. For example, say "pull request" instead of PR", since only a subset of Tweet readers will be active GitHub users.
```
Comment (jasonblais) [inline source/process/marketing-guidelines.rst]: PR" --> "PR"; 
Tweet readers --> tweet readers; 
will be --> are

File: source/process/marketing-guidelines.rst [hunk 7]
```diff
@@ -63,3 +65,15 @@ Guidelines: 12 tips for sparkling content
...
+
+
+Twitter Guidelines 
+==============================================================
+
+Composing Tweets: 
+
+1. **Avoid acronyms** - Avoid acronyms when possible. For example, say "pull request" instead of PR", since only a subset of Tweet readers will be active GitHub users. 
+2. **Use the active voice** - Avoid "has", "was", "have been" when possible. For example, instead of "Hackfest has started!" say "Hackfest starts now!" 
+3. **Include no more than one link"** - Include no more than one link per tweet, and place it close to the end of the tweet.
```
Comment (jasonblais) [inline source/process/marketing-guidelines.rst]: Maybe we can say "Include at most one link"?

File: source/process/marketing-guidelines.rst [hunk 8]
```diff
@@ -31,14 +33,14 @@ Principles
...
- 3. Part of our brand. Our style and message must be consistent over the long term. Therefore, all of our content, irrespective of purpose or subject-matter, must pull the reader in the same direction and reinforce our key messages. 
+ 1. **Magnetic to IT.** We need headlines and openings that draw in IT audiences (these include Linux admins, security experts, directors of end-user computing, CIOs). Bear in mind that, although they work in IT, these audiences are typically more interested in the business benefits of technology than technical features. 
+ 2. **Appealing to secondary audiences.** These include open source and in-house developers and end users at highly regulated companies.
+ 3. **Part of our brand.** Our style and message must be consistent over the long term. Therefore, all of our content, irrespective of purpose or subject-matter, must pull the reader in the same direction and reinforce our key messages. 
  4. High quality. Quality in writing reflects the professionalism of our organization. We pay attention to the rules of grammar and spelling.
```
Comment (jasonblais) [inline source/process/marketing-guidelines.rst]: Add bolding for "High Quality" too

File: source/process/marketing-guidelines.rst [hunk 9]
```diff
@@ -63,3 +65,15 @@ Guidelines: 12 tips for sparkling content
  10. **Edit ruthlessly**. Use the first draft to get your thoughts together. Then leave it for a few hours (or even days). Shorten, delete, and rewrite anything that does not add to the meaning. Run a spelling and grammar check.
  11. **Ask a colleague**. Still not 100% happy? A colleague should review the text and tell you if anything is unclear or if you could express yourself better. They might offer some useful fresh insights.
  12. **Now, relax**. If you are in the wrong frame of mind you will either get writer's block or your writing will be stiff and unnatural. Take your time. Remove any distractions. Keep things simple. 
+
+
+Twitter Guidelines
```
Comment (jasonblais) [inline source/process/marketing-guidelines.rst]: The section headings aren't really consistent.. one is `How to write purposefully and persuasively`, and another is `Twitter Guidelines`

**Comment by lfbrock** [comment]
+1

**Pull request merged**